_pkgdown.yml

```diff
@@ -0,0 +1,14 @@
+url: https://baselinenowcast.epinowcast.org/
+template:
+  math-rendering: mathjax
+
+development:
+  mode: auto
+
+navbar:
+  structure:
```

Comment:
I was expecting to see the vignette plan here but no issue adding that in at a later date.

Reply:
I started on the vignette in a separate PR, I didn't understand a lot of this and figured we'd start with something pretty simple (a Quick Start with the package name) and can build up from there

Reply:
I can make an issue to outline planned vignettes?